app/main.py

```diff
@@ -2,0 +1,30 @@
+import os
+
+
+def move_file(command: str) -> None:
+
+    mv, source_file, destination = command.split()
+    if mv != "mv":
+        return
+
+    if "/" in destination:
+        if destination[-1] == "/":
+            destination += source_file
```

Comment:
Appending `source_file` to `destination` when `destination` ends with a '/' is not sufficient. This logic assumes that the destination directory already exists, which may not be the case. Consider checking if the directory exists and creating it if necessary.